src/settings-tab.ts

```diff
@@ -27,30 +26,40 @@ export class SettingsTab extends PluginSettingTab {
 		containerEl.empty();
 
 		new Setting(containerEl)
-			.setName('Day One files')
-			.setDesc('Location where you extracted the zip file from Day One')
-			.addText((text) =>
-				text
-					.setPlaceholder('Directory')
+			.setName('Day One import folder')
+			.setDesc(
+				`Folder where Day One JSON exports are located.
+				All JSON files in this folder will be imported unless a specific file is set below.`
+			)
+			.addSearch((cb) => {
+				cb.setPlaceholder('Example: folder1/folder2')
 					.setValue(this.plugin.settings.inDirectory)
-					.onChange(async (value) => {
-						this.plugin.settings.inDirectory = normalizePath(value);
+					.onChange(async (importFolder) => {
+						importFolder = importFolder.trim();
+						importFolder = importFolder.replace(/\/$/, '');
```

Comment:
We should keep using `normalizePath` here rather than doing manually replacements https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+%60normalizePath()%60+to+clean+up+user-defined+paths